src/Razor/src/Microsoft.AspNetCore.Razor.Common/Telemetry/TelemetryReporter.cs

```diff
@@ -73,4 +118,47 @@ private void Report(TelemetryEvent telemetryEvent)
             _logger?.LogError(e, "Failed logging telemetry event");
         }
     }
+
+    private static string GetDescription(Exception exception)
+    {
+        const string CodeAnalysisNamespace = nameof(Microsoft) + "." + nameof(CodeAnalysis);
+        const string AspNetCoreNamespace = nameof(Microsoft) + "." + nameof(AspNetCoreNamespace);
```

Comment:
So we're looking for types that start with `Microsoft.AspNetCoreNamespace`? I would have thought we wanted `Microsoft.AspNetCore.`.